### PR TITLE
Docker runtime

### PR DESCRIPTION
**Pull request by Koufan-De-King**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VITE_BACKEND_URL="https://mynameis"
+VITE_FRONTEND_URL="https://mynameisnot
```
Comment (stephane-segning) [inline .env.example:2]: Why do you need the frontend url?

Comment (Koufan-De-King) [reply .env.example:2]: Oops, forgot it while testing for multiple variables. Will remove it!

File: src/shared/projectEnvVariables.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+type ProjectEnvVariablesType = Pick<
+  ImportMetaEnv,
+  "VITE_BACKEND_URL" | "VITE_OTHER_ENV_VARIABLE"
+>;
+
+const projectEnvVariables: ProjectEnvVariablesType = {
+  VITE_BACKEND_URL: "${VITE_BACKEND_URL}",
+  VITE_OTHER_ENV_VARIABLE: "${VITE_OTHER_ENV_VARIABLE}",
```
Comment (stephane-segning) [inline src/shared/projectEnvVariables.ts:8]: What is this `OTHER_ENV_VARIABLE`?

File: src/shared/projectEnvVariables.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export const getProjectEnvVariables = (): ProjectEnvVariables => {
+  return {
+    envVariables: {
+      VITE_BACKEND_URL: !projectEnvVariables.VITE_BACKEND_URL.includes("VITE_")
+        ? projectEnvVariables.VITE_BACKEND_URL
+        : import.meta.env.VITE_BACKEND_URL,
+      VITE_OTHER_ENV_VARIABLE:
+        !projectEnvVariables.VITE_OTHER_ENV_VARIABLE.includes("VITE_")
+          ? projectEnvVariables.VITE_OTHER_ENV_VARIABLE
+          : import.meta.env.VITE_OTHER_ENV_VARIABLE,
```
Comment (stephane-segning) [inline src/shared/projectEnvVariables.ts:24]: `OTHER_ENV_VARIABLE` is not use, isn't it? Please remove it.

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: src/shared/projectEnvVariables.ts (hunk 3 above)
Comment (Koufan-De-King) [reply src/shared/projectEnvVariables.ts:24]: Okay. I will.

**Review by stephane-segning — DISMISSED** [review]

**Review by NkwaTambe — APPROVED** [review]

**Pull request merged**